app.js

```diff
@@ -104,13 +104,30 @@ function calculatorDisplay(value) {
       getData("9");
       break;
     case "decimal":
-      getData(".");
+      if (decimalCount === 0) {
+        getData(".");
+        decimalCount++;
+      }
```

Comment:
You're using an integer to represent decimal count, but it can only ever be 0 or 1, so it might be worth to simply use true/false